### PR TITLE
Remove 4.2

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

* Remove Kiwi profiles for 4.1 and 4.2
* Other small removals for 4.2

## Links

Ports:
* 4.3: https://github.com/SUSE/spacewalk/pull/22948

## Changelogs

- [x] No changelog needed

### CODE REVIEW

File: testsuite/ext-tools/maintenance_json_generator.py [hunk 1]
```diff
@@ -108,7 +93,7 @@ def parse_args():
         description="This script reads the open qam-manager requests and creates a json file that can be fed in the "
                     "BV testsuite pipeline")
     parser.add_argument("-v", "--version", dest="version",
-                        help="Version of SUMA you want to run this script for, options are 42 for 4.2 or 43 for 4.3",
+                        help="Version of SUMA you want to run this script for, options are 43 for 4.3",
```
Comment (ktsamis) [inline testsuite/ext-tools/maintenance_json_generator.py:96]: maybe change temporarily the message? 
```suggestion
                        help="Version of SUMA you want to run this script for, the only option for now is 43 for 4.3",
```

Comment (ktsamis) [reply testsuite/ext-tools/maintenance_json_generator.py:96]: I say temporarily, I mean until 5.0 comes out.

Comment (Bischoff) [reply testsuite/ext-tools/maintenance_json_generator.py:96]: Yup, good idea

Comment (Bischoff) [reply testsuite/ext-tools/maintenance_json_generator.py]: Done

**Review by ktsamis — APPROVED** [review]

**Review by nodeg — APPROVED** [review]
LGTM. I like the new raises when using older images.

**Review by srbarrios — APPROVED** [review]

**Review by srbarrios — APPROVED** [review]

**Pull request merged**